setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
```

Comment:
Ideally this would follow the work currently being done by @dirk-thomas to be able to rosrun that node: (c.f. https://github.com/ros2/demos/pull/136/files#diff-ed7e363075493ff359e5d240b77a9ec6R1)